internal/util/debug/debug.go

```diff
@@ -63,9 +63,10 @@ type Handler struct {
 //
 //nolint:vet // for readability
 type ListenOpts struct {
-	TCPAddr string
-	L       *zap.Logger
-	R       prometheus.Registerer
+	TCPAddr         string
+	L               *zap.Logger
+	R               prometheus.Registerer
+	FerretdbStarted <-chan struct{}
```

Comment:
Since we will need atomic for other probes, let's use atomic there too.

Also, the "FerretDB" prefix is excess. On the one hand, what else could it be? On the other hand – it does not even matter because the debug handler only knows about the probe that happened to be called "started". There will be no different logic for other probes.